lib/ansible/modules/cloud/vmware/vmware_vm_shell.py

```diff
@@ -30,13 +30,11 @@
 options:
     datacenter:
         description:
-            - The datacenter hosting the VM
-            - Will help speed up search
-        required: False
-        default: None
+            - The datacenter hosting the Virtual Machine.
+            - If set, it will help to speed up virtual machine search.
     cluster:
         description:
-            - The cluster hosting the VM
+            - The cluster hosting the virtual machine
             - Will help speed up search
         required: False
```

Comment:
Values that are not required, don't need this. Also default null values are obsolete.